### PR TITLE
Fish component

### PR DESCRIPTION
**Pull request by edkotkas**

### CODE REVIEW

**Review by Zattri — COMMENTED** [review]
Original fish image is now too small, pls fix

File: src/app/app.component.html [hunk 1]
```diff
@@ -31,15 +17,20 @@
     </div>
 
     <div class="centreText centreTop catchText">
+      <div *ngIf="!casted && !fish">Cast your line!</div>
```
Comment (Zattri) [inline src/app/app.component.html]: Won't this have a problem later if we reset the fish or something? - Rename !fish to something else, or set a flag upon game start and check that?

File: src/app/fish/fish.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@Component({
+  selector: 'sf-fish',
+  templateUrl: './fish.component.html',
+  styleUrls: ['./fish.component.scss']
+})
+export class FishComponent implements OnInit {
+
+  @Input() fish;
+  public uncaught = ['fish', 'centreMiddle', 'animated', 'bounceInUp'];
+  public caught = ['fish', 'centreMiddle', 'animated', 'shake'];
```
Comment (Zattri) [inline src/app/fish/fish.component.ts]: Add in more fish catching animations here?

Comment (edkotkas) [reply src/app/fish/fish.component.ts]: yeah can do with more of them, need to make this part better in general as there's duplication

File: src/app/app.component.html (hunk 1 above)
Comment (edkotkas) [reply src/app/app.component.html]: shouldn't unless you're doing something funky. possibly have a settings like "firstRun"

**Pull request merged**